File: arithmetic.py
from array import *
age=array('i',[])
sum=0
first_age=int(input("Enter your first age: "))
common_difference=int(input("Enter the common difference: "))
participants=int(input("Enter the number of participants: "))
for i in range(participants):
    age.append(first_age+i*common_difference)
    sum=sum+age[i]
    print("The ages of paticipant :",i+1," is", age[i],"\n")
average=sum/participants
print("The average age of the participants is: ",average)

File: arithmetic.java
import java.util.Scanner;

/**
 * arithmetic
 */
public class arithmetic {

    public static void main(String[] args) {
        //a program that accepts the starting age from a group of people value,common diffrence and the number of terms and prints the series of the series.
        int start, diff, n;
        Scanner entry = new Scanner(System.in);
        System.out.println("Enter the starting age: ");
        start = entry.nextInt();
        System.out.println("Enter the common diffrence: ");
        diff = entry.nextInt();
        System.out.println("Enter the number of people: ");
        n = entry.nextInt();
        // the ages of the people are stored in an array
        int[] ages = new int[n];
        for (int i = 0; i < n; i++) {
            ages[i] = start + i * diff;
            System.out.printf( "The ages of %d is %d\n",i+1,ages[i]);
        }
    
        
        

    }
}